resources/public/api/conf/1.0/schemas/lossClaims/amendClaim/amend_request.json

```diff
@@ -6,7 +6,7 @@
   "properties": {
     "typeOfClaim": {
       "type": "string",
-      "description": "The type of claim to be applied. A self employment loss claim can only be of type carry-forward or carry-sideways",
+      "description": "The type of claim to be applied.  Permitted values for \"self-employment\": carry-forward, carry-sideways.  Permitted values for \"uk-property-non-fhl\" and \"foreign-property\": carry-sideways, carry-sideways-fhl, carry-forward-to-carry-sideways.",
```

Comment:
mentions foreign property here, afaik we don't have foreign property as a type of loss in V1?

Reply:
Good spot, BA has confirmed that references to foreign-property can be removed from v1 - I'll do that now.